src/OidcClientSettings.ts

```diff
@@ -142,6 +142,12 @@ export interface OidcClientSettings {
      * Defines request timeouts globally across all requests made to the authorisation server
      */
     requestTimeoutInSeconds?: number | undefined;
+
+    /**
+     * https://datatracker.ietf.org/doc/html/rfc6749#section-3.3 describes behavior when omitting scopes from sign in requests
+     * If the IDP supports default scopes, this setting will ignore the scopes property passed to the config
```

Comment:
please add: (Default: false) at the end of the comment, see other comments...

Reply:
updated